src/_data/survey-vers.json

```diff
@@ -0,0 +1,7 @@
+{
+  "uniqueId": "2020Q1",
+  "startMsEpoch": "2020/03/03 9:00",
```

Comment:
`startDate`?

And, I'd make sure that the format here aligns with something in https://api.dart.dev/stable/2.7.1/dart-core/DateTime/parse.html.

Perhaps just ensure that the value always looks like `"2012-02-27 13:27:00"`?